src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -2310,6 +2310,14 @@
        -s https://github.com/JohnDoe/my-first-static-web-app -l WestUs2 -b master
 """
 
+helps['staticwebapp update'] = """
+    type: command
+    short-summary: Update a static app. Return the app updated.
+    examples:
+    - name: Update static app in a subscription.
+      text: az staticwebapp update -n MyStaticAppName -g MyExistingRg -b master
```

Comment:
do we need the branch to update the static web app?

Reply:
no, we don't. will revise this example. but wondering do we allow the branch to be updated?